### PR TITLE
📚 Docs Refresh: Google Ads Source

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Updated language and clarity.
- Reorganized sections for efficient consumption.
- Added Airbyte Cloud setup instructions.

### CODE REVIEW

File: docs/integrations/sources/google-ads.md [hunk 1]
```diff
@@ -4,44 +4,42 @@
...
-| Replicate Incremental Deletes | No |
-| SSL connection | Yes |
-| Namespaces | No |
-
-### Performance considerations
-
-This source is constrained by whatever API limits are set for the Google Ads that is used. You can read more about those limits in the [google developer docs](https://developers.google.com/google-ads/api/docs/best-practices/quotas).
+1. Get the customer ID for your account. Learn how to do that [here](https://support.google.com/google-ads/answer/1704344)
+2. If your access to the account is through a manager account, get the customer ID of the manager account.
+3. Fill out a start date, conversion window, and custom [GAQL](https://developers.google.com/google-ads/api/docs/query/overview).
```
Comment (sherifnada) [inline docs/integrations/sources/google-ads.md:37]:
```suggestion
3. Fill out a start date, and optionally, a conversion window, and custom [GAQL](https://developers.google.com/google-ads/api/docs/query/overview).
```

File: docs/integrations/sources/google-ads.md [hunk 2]
```diff
@@ -4,44 +4,42 @@
...
-| Full Refresh Sync | Yes |
-| Incremental Sync | Yes |
-| Replicate Incremental Deletes | No |
-| SSL connection | Yes |
-| Namespaces | No |
-
-### Performance considerations
-
-This source is constrained by whatever API limits are set for the Google Ads that is used. You can read more about those limits in the [google developer docs](https://developers.google.com/google-ads/api/docs/best-practices/quotas).
+1. Get the customer ID for your account. Learn how to do that [here](https://support.google.com/google-ads/answer/1704344)
```
Comment (sherifnada) [inline docs/integrations/sources/google-ads.md:35]: there should also be a step saying to select `Sign in via Google` and clicking the authorize button

File: docs/integrations/sources/google-ads.md [hunk 3]
```diff
@@ -79,10 +77,15 @@ When you apply for a token, you need to mention:
 
 If for any reason the request gets denied, let us know and we will be able to unblock you.
 
-## Understanding Google Ads Query Language
+#### Understanding Google Ads Query Language
 
 The Google Ads Query Language can query the Google Ads API. Check out [Google Ads Query Language](https://developers.google.com/google-ads/api/docs/query/overview)
 
+## Rate Limiting & Performance Considerations
```
Comment (sherifnada) [inline docs/integrations/sources/google-ads.md:84]:
```suggestion
## Rate Limiting & Performance Considerations (Airbyte Open Source)
```

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/google-ads.md [hunk 4]
```diff
@@ -79,10 +77,15 @@ When you apply for a token, you need to mention:
...
+
+This source is constrained by whatever API limits are set for the Google Ads that is used. You can read more about those limits in the [google developer docs](https://developers.google.com/google-ads/api/docs/best-practices/quotas).
```
Comment (marcosmarxm) [inline docs/integrations/sources/google-ads.md:86]:
```suggestion
This source is constrained by whatever API limits are set for the Google Ads that is used. You can read more about those limits in the [Google Developer docs](https://developers.google.com/google-ads/api/docs/best-practices/quotas).
```

**Review by marcosmarxm — APPROVED** [review]
LGTM small comment

File: docs/integrations/sources/google-ads.md (hunk 2 above)
Comment (avaidyanatha) [reply docs/integrations/sources/google-ads.md:35]: Ah. yes! Done.

**Pull request merged**